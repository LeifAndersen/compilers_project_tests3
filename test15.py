a and b or 3 or not b and (d or c or b and (c or d) and w)

