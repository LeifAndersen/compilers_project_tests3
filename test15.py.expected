(program
 (or (and (get-global a) (get-global b))
     3
     (and (not (get-global b))
          (or (get-global d)
              (get-global c)
              (and (get-global b)
                   (or (get-global c) (get-global d))
                   (get-global w))))))

